docs/release-procedure.md

```diff
@@ -59,14 +56,23 @@ Push the commit, make a PR _against `release-X.Y.Z`_ branch and, when it's merge
   * Push the tag to master: `git push upstream X.Y.Z`
     * If you're re-pushing a tag (beware! you shouldn't be doing that, no, _really_!), you can remove remote tag and push a local one by `git push upstream :refs/tags/X.Y.Z`
 
+***Note***: We are currently working on automating the release notes procedure - check 
+[#4747](https://github.com/modin-project/modin/issues/4747) for updates!
```

Comment:
Should we really add this? If so, please create an issue to update this phrase later in order not to forget to do this.

Reply:
I'm open to changing it - @mvashishtha recommended having it here for visibility!